admin/database/postgres/migrations/0032.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE usergroups ADD COLUMN description TEXT;
```

Comment:
Consider `NOT NULL DEFAULT ''` (since it makes the code simpler and there is no meaningful difference between a `null` value and empty string in this case).